sandbox/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Using lightweight alpine image
+FROM python:3.10-alpine
```

Comment:
https://pythonspeed.com/articles/alpine-docker-python/

May be better to try `python:3.10-slim`